### ISSUE TITLE
Getting back to list of repos is hard

### ISSUE THREAD

**Issue by youpdidou** [FundersClub/fire#68]
Got this feedback from a friend, and I agree!

"from here it's hard to figure out how to get back to Repos"

![image](https://cloud.githubusercontent.com/assets/702045/23763231/96c33c6a-04ad-11e7-999d-c11bb2669ac1.png)

**Comment by Beamanator** [comment]
Yeah, that would be a nice feature to add - a link back to the repo
